mmv1/templates/terraform/custom_delete/firebase_app_check_service_config.go

```diff
@@ -0,0 +1,40 @@
+var obj map[string]interface{}
```

Comment:
We should note in the resource docs that deleting the resource disables the enforcement mode!